fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerTest.java

```diff
@@ -92,6 +96,37 @@ public IndexMaintenanceFilter getIndexMaintenanceFilter() {
         }
     }
 
+    protected void disableAll(List<Index> indexes) {
+        try (FDBRecordContext context = openContext()) {
+            // disable all
+            for (Index index : indexes) {
+                recordStore.markIndexDisabled(index).join();
+            }
+            context.commit();
+        }
+    }
+
+    protected void validateIndexes(List<Index> indexes) {
+        final FDBStoreTimer timer = new FDBStoreTimer();
+        for (Index index: indexes) {
+            if (index.getType().equals(IndexTypes.VALUE)) {
+                try (OnlineIndexScrubber indexScrubber = OnlineIndexScrubber.newBuilder()
+                        .setDatabase(fdb).setMetaData(metaData).setIndex(index).setSubspace(subspace)
```

Comment:
I believe this can use `newScrubberBuilder()` to fill in most of the defaults for some of those parameters.

Reply:
✅